subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/dependencymanagement/SamplesTroubleshootingDependencyResolutionIntegrationTest.groovy

```diff
@@ -25,28 +26,34 @@ class SamplesTroubleshootingDependencyResolutionIntegrationTest extends Abstract
 
     private static final String COPY_LIBS_TASK_NAME = 'copyLibs'
 
+    File repositoryMirrorScript = RepoScriptBlockUtil.createMirrorInitScript()
+
+    def setup() {
+        executer.usingInitScript(repositoryMirrorScript)
+    }
+
     @Rule
     Sample sample = new Sample(testDirectoryProvider)
 
     @UsesSample("userguide/dependencyManagement/troubleshooting/cache/changing")
-    def "can declare custom TTL for dependency with dynamic version"() {
+    def "can declare custom TTL for dependency with changing version"() {
         executer.inDirectory(sample.dir)
 
         when:
         succeeds(COPY_LIBS_TASK_NAME)
 
         then:
-        sample.dir.file('build/libs/spring-web-5.0.3.BUILD-SNAPSHOT.jar').isFile()
+        sample.dir.file('build/libs/slf4j-api-1.7.25.jar').isFile()
     }
 
-    @UsesSample("userguide/dependencyManagement/troubleshooting/cache/changing")
-    def "can declare custom TTL for dependency with changing version"() {
+    @UsesSample("userguide/dependencyManagement/troubleshooting/cache/dynamic")
+    def "can declare custom TTL for dependency with dynamic version"() {
         executer.inDirectory(sample.dir)
 
         when:
         succeeds(COPY_LIBS_TASK_NAME)
 
         then:
-        sample.dir.file('build/libs').listFiles().any { it.name.startsWith('spring-web-5.') }
+        sample.dir.file('build/libs').listFiles().any { it.name.startsWith('slf4j-api-1.7.') }
```

Comment:
I don't think this test would pass. It's still using `org.springframework:spring-web:5.+` on `master`: https://github.com/gradle/gradle/blob/master/subprojects/docs/src/samples/userguide/dependencyManagement/troubleshooting/cache/dynamic/build.gradle#L8

Reply:
That's a forgotten revert indeed - fixing